docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+
+By default, there will be separate Pull Requests per dependency, and major updates will be separated out from non-major.
+
+### Package managers with lock files
+
+Many package managers today support "lock files", which serve to freeze the entire dependency tree including transitive dependencies.
+Such managers include npm, Yarn, Bundler, Composer, Poetry, Pipenv, and Cargo.
+
+When lock files exist, it's essential that any changes to package files are accompanied with a compatible change to the associated lock file.
+Although Renovate can patch/update package files directly, a package manager's lock file is typically too complex to "reverse engineer" so therefore Renovate will rely on the package manager itself to perform the lock file update.
+Here is a simplified example:
+
+- The repository has a `package.json` and `package-lock.json` containing version `1.0.0` of a dependency
+- Renovate determines that version `1.1.0` is available
+- Renovate patches the `package.json` to change the dependency's version from `1.0.0` to `1.1.0`
+- Renovate then runs `npm install`, which triggers `npm` to update the `package-lock.json` accordingly
+- Renovate then commits both the `package.json` and `package-lock.json` files together for the PR
+
+### Custom dependency extraction
+
+Renovate supports 60+ different types of package files natively, but sometimes dependencies will not be detected by Renovate by default due to either:
+
+- The package manager/file format is not yet supported, or
+- The file format is not a standard or is completely proprietary
+
+In such cases, Renovate has a "regex" manager which you can configure with custom patterns to extract dependencies regardless.
+Users can configure the regex manager by telling it:
+
+- Which file pattern(s) to match
+- How to identify the dependency name and version from within the file
+- Which datasource (e.g. Docker registry, npm registry, etc) to use to look up new versions
+
+The end result is that Renovate can keep dependencies in custom file formats up-to-date as long as the dependency datasource is already known to Renovate.
+
+## DevOps tooling
+
+Renovate is increasingly used for purposes which are traditionally described as DevOps instead of Developer.
+
+### DevOps / Infrastructure as Code updates
+
+Repositories today consist of more than just development dependencies, and commonly include DevOps-related files like CI/CD configs or "Infrastructure as Code" (IaC) files like Docker, Kubernetes or Terraform files.
+Renovate considers these all to be forms of "package managers" and "package files" and therefore detects and updates them accordingly.
+
+Docker-compatible images are one of the key building blocks of modern software and so are most commonly found, in both CI/CD pipeline configs as well as referenced in IaC files.
+Renovate will detect these then look up tags and digests from Docker registries to determine not only if a new tag exists but also if a new digest does too.
```

Comment:
> Renovate will detect these then look up tags and digests from Docker registries to determine not only if a new tag exists but also if a new digest does too.

I keep re-reading this sentence, it's tricky to understand.

Maybe something like this would work better? Feel free to edit the suggestion if needed!

```suggestion
Renovate will detect these IaC files and looks up tags and digests from Docker registries to determine if a new tag exists, Renovate will also determine if a new digest exists.
```

Reply:
YES! That's way better, thank you! 😄